certs/certs.go

```diff
@@ -30,8 +30,18 @@ import (
 )
 
 const (
-	rsaKeySize   = 2048
 	duration365d = time.Hour * 24 * 365
+	rsaKeySize   = 2048
+	tlsKey       = "tls.key"
+	tlsCrt       = "tls.crt"
+	//ClusterCaName is the secret name suffix for apiserver CA
+	ClusterCaName = "ca"
+	//EtcdCaName is the secret name suffix for the Etcd CA
+	EtcdCaName = "etcd"
+	//ServiceAccountName is the secret name suffix for the Service Account keys
+	ServiceAccountName = "sa"
+	//FrontProxyName is the secret name suffix for Front Proxy CA
+	FrontProxyName = "proxy"
```

Comment:
Should this be `FrontProxyCaName` to go with the naming convention of the other CA vars?